ford/sourceform.py

```diff
@@ -2532,3 +2537,50 @@ def get_name(self,item):
             return name
 
 namelist = NameSelector()
+
+
+class ExtModule(FortranModule):
+
+    def __init__(self):
+        self.name = ''
+        self.uses = []
+        self.pub_procs = {}
+        self.pub_absints = {}
+        self.pub_types = {}
+        self.pub_vars = {}
+        self.extURL = ''
```

Comment:
Rename suggestion for this variable: `extURL` -> `external_URL`

Reply:
Fine by me.